account_page.php

```diff
@@ -86,12 +86,19 @@
 
 html_page_top( lang_get( 'account_link' ) );
 
+$t_user_id = auth_get_current_user_id();
+
 # extracts the user information for the currently logged in user
 # and prefixes it with u_
-$t_row = user_get_row( auth_get_current_user_id() );
+$t_row = user_get_row( $t_user_id );
 
 extract( $t_row, EXTR_PREFIX_ALL, 'u' );
 
+if ( is_blank( $u_token ) ) {
+	$u_token = helper_generate_uuid();
+	user_set_field( $t_user_id, 'token', $u_token );
```

Comment:
How is this different from the cookies that are stored unencrypted?  The password is hashed because it can be re-used elsewhere and that we never need to display it.  That is not the case with token and cookie.